### PR TITLE
pushed

### PR DESCRIPTION
**Pull request by Daniekeys**
#### PR Checklist

- [ ] `npm run lint` passes

### CODE REVIEW

File: assets/svg/index.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export const Bars = () => {
```
Comment (sammy-code98) [inline assets/svg/index.tsx:1]: please can you use a descriptive name for this file

Or even better separate  them into two components ie bars.tsx and times.tsx files

File: assets/svg/index.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    return (
+        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" className="w-6 h-6">
+            <path stroke-linecap="round" stroke-linejoin="round" d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5" />
+        </svg>
+
+
+    )
+}
+
+export const Times = () => {
```
Comment (sammy-code98) [inline assets/svg/index.tsx:11]: export const Times = ():JSX.Element => {....}

add typescript types

**Review by sammy-code98 — CHANGES_REQUESTED** [review]

**Comment by sammy-code98** [comment]
image

Use a drawer for mobile navigation , to prevent hero content being pushed down the  screen when nav is triggered

**Comment by sammy-code98** [comment]
image

hamburger button should be placed at the far right, after the internalization icon

File: assets/svg/index.tsx (hunk 1 above)
Comment (Daniekeys) [reply assets/svg/index.tsx:1]: so for icons stuff like this its cool to have it exported and mutliple in one place. there wont be issues at all with that method.

Comment (sammy-code98) [reply assets/svg/index.tsx:1]: > so for icons stuff like this its cool to have it exported and mutliple in one place. there wont be issues at all with that method.

yes sire, that way it makes the component more reusable..

File: assets/svg/index.tsx (hunk 2 above)
Comment (Daniekeys) [reply assets/svg/index.tsx]: @sammy-code98  done

**Review by sammy-code98 — APPROVED** [review]

**Review by chimdie — CHANGES_REQUESTED** [review]

**Review by chimdie — APPROVED** [review]

**Pull request merged**